docs/usage/filters.md

```diff
@@ -0,0 +1,291 @@
+# Filter knowledge
+
+In OpenCTI, you can filter data to target or view entities that have particular attributes.
+
+## Filters usages
+Filters are used in different locations in the platform.
+- in entities lists: filters restrict the displayed entities to the ones respecting the chosen filtering. If an export or a background task is generated, only the filtered data will be taken into account.
+- in investigations and knowledge graphs: filters restrict the displayed entities to the ones respecting the chosen filtering.
+- in workspaces: filters enable to create graphs with only the wanted entities.
+- in feeds, taxii collections, triggers, streams, playbooks: filters restrict the data to manage or the events to listen to.
+
+There are two types of filters:
+- dynamic filters: They are not stored in the database, they enable to filter view in the UI. Exemples: filters in entities list, investigations, knowledge graphs;
+- stored filters: They are attributes of an entity, they are stored as they other attributes. Exemples: filters of workspaces, feeds, taxii collections, triggers, streams, playbooks.
+
+## Filters format (since 5.12)
+The filters format since OpenCTI 5.12 is of type FilterGroup (see below). This is the format to use when doing query via the API.
+This enables to do complex filters imbrication with different operators ('and', 'or').
+
+```
+FilterGroup: {
+    mode: string, // 'and' or 'or'
+    filters: Filter[],
+    filterGroups: FilterGroup[],
+}
+
+Filter {
+    key: string,
+    values: string[],
+    operator: string,
+    mode: string, // 'and' or 'or'
+}
+```
+
+In a filter group, the 'mode' can be equal to 'and' or 'or' and represent the operator between the different filters and groups in the 'filters' and 'filterGroups' arrays.
+The 'filters' and 'filterGroups' arrays are composed of objects of type Filter/FilterGroup.
+The base entity of a filter group object is a filter with 4 attributes:
+- a key, representing on what we want to do the filtering (exemple: 'objectLabel' to filter on labels, 'createdBy' to filter on the entity creation date, 'report_types', 'entity_type'... ).
```

Comment:
if we want a complete API documentation, we need an exhaustive list. Some if these keys are not trivial 😉

Reply:
I'm gonna list some important keys + keys with a special behavior